CommonLibSF/CMakeLists.txt

```diff
@@ -0,0 +1,166 @@
+cmake_minimum_required(VERSION 3.21)
```

Comment:
bump CMake? 3.21 is pretty ancient

Reply:
Sure, I haven't updated cmake in a while, by the way sfse uses 3.18 :nerd: